gradle-module-plugin/README.md

```diff
@@ -28,16 +28,23 @@ The easiest way to get started with this plugin is to create a new module projec
    | keystorePassword  | gradlew signModule --keystorePassword=mysecret  | ignition.signing.keystoreFile=mysecret  |
 
 
-4. When depending on artifacts (dependencies) from the Ignition SDK, they should be specified as `compileOnly` dependencies as they will be provided by the Ignition platform at runtime.  Dependencies that are applied with either the `modlApi` or `modlImplementation` _Configuration_ in any subproject of your module will be collected and included in the final modl file.
-   Test and Compile-time dependencies should be specified in accordance with the best practices described in Gradle's `java-library` [plugin documentation](https://docs.gradle.org/current/userguide/java_library_plugin.html).  See the Dependency Configuration section below for more info on configuring and managing dependencies.
+4. When depending on artifacts (dependencies) from the Ignition SDK, they should be specified as `compileOnly` or `compileOnlyApi`  dependencies as they will be provided by the Ignition platform at runtime.  Dependencies that are applied with either the `modlApi` or `modlImplementation` _Configuration_ in any subproject of your module will be collected and included in the final modl file, including transitive dependencies. Test and Compile-time dependencies should be specified in accordance with the best practices described in Gradle's `java-library` [plugin documentation](https://docs.gradle.org/current/userguide/java_library_plugin.html). This plugin diverges from Gradle's `java-library` semantics only where we resolve dependencies for inclusion into the modl.  Unlike the `java-library` plugin, transitive dependencies of `modlImplementation` configurations WILL be collected for use in the module. That one difference aside, all other implications apply as documented by the Gradle java-library plugin (e.g. - publishing, artifact uploading, transitive dependency handling, etc).  Test-only dependencies should NOT be marked with any `modl` configuration.
```

Comment:
Already ping'd you about this, but the distinction between `modlImplementation`s and `implementation`s handling of transitives seems to just muddy the water since implementation includes transitives on the runtime classpath which is analogous to what we are doing here.